app/src/main/java/com/github/bgabriel998/softwaredevproject/ProfileActivity.java

```diff
@@ -94,8 +94,8 @@ public void submitUsernameButton(View view) {
         Log.d("CURRENT_USERNAME", "onSubmit: " + currentUsername);
         if(username.equals(currentUsername)) {
             Log.d("CURRENT_USERNAME", "onSubmit: EQUAL_CASE");
-            Toast toast = Toast.makeText(getApplicationContext(), "You can't choose the username you already have!", Toast.LENGTH_SHORT);
-            toast.show();
+            Snackbar snackbar = Snackbar.make(findViewById(android.R.id.content), "You can't choose the username you already have!" , Snackbar.LENGTH_LONG);
```

Comment:
Should the message be a constant instead of a magic string?

Reply:
Good point. Next week all these magic strings will be on the string.xml file!